cmd/kubeadm/app/cmd/join.go

```diff
@@ -57,12 +58,27 @@ func NewCmdJoin(out io.Writer) *cobra.Command {
 		"(required) Shared secret used to secure bootstrap. Must match the output of 'kubeadm init'",
 	)
 
+	cmd.PersistentFlags().BoolVar(
+		&skipPreFlight, "skip-preflight-checks", false,
```

Comment:
New flags must be added to `hack/verify-flags/known-flags.txt`, otherwise build won't pass.